### DISCUSSION TITLE
Pin Title printing undefined instead of board title

### DISCUSSION THREAD

**Discussion by robertncole1** [nss-evening-cohort-14/evening-client-side#169] [Help Me!] [answered]
Have you searched the other help tickets?
Yes

Describe your issue:
When I click on pins, the title for info appears but the title is undefined instead of the boardTitle that is being plugged into the innerHTML.

List any error messages that appear
None

What have you googled?
https://stackoverflow.com/questions/17546953/cant-access-object-property-even-though-it-shows-up-in-a-console-log
honestly, not sure how to word it

What have you tried? Tell us everything you have tried.
I've tried to edit my dom events to make sure its pulling the right thing but that didn't work.
I tried editing the boardObject but that didn't work.

Links to my code:
BoardInfo
https://github.com/robertncole1/ASSIGNMENT-Firebase-Pinterest/blob/main/src/javascripts/components/boardInfo.js
Dom Events:
https://github.com/robertncole1/ASSIGNMENT-Firebase-Pinterest/blob/main/src/javascripts/components/events/domEvents.js

![Screen Shot 2021-03-08 at 8 38 16 PM](https://user-images.githubusercontent.com/76854545/110410260-392d3300-804e-11eb-9429-220b5838d62e.png)

**Comment by TrinityChristiana** [comment]
`boardInfo` needs to be passed `boardInfoObject.board`

This line: https://github.com/robertncole1/ASSIGNMENT-Firebase-Pinterest/blob/main/src/javascripts/components/events/domEvents.js#L11

**Comment by robertncole1** [comment, marked as answer]
So in my getSingleBoard function, my dbUrl was calling my pins.json instead of my boards.json so it wasn't finding the board title when calling the URL 🥴
